packages/report-server/src/reportBuilder/transform/aliases/periodConversionAliases.ts

```diff
@@ -13,3 +13,13 @@ export const convertPeriodToWeek = () =>
       '...': '*',
     },
   ]);
+
+export const convertEventDateToWeek = () =>
+  buildTransform([
+    {
+      transform: 'select',
+      "'period'": "convertToPeriod($row.eventDate, 'WEEK')",
```

Comment:
Man, it's kinda annoying that data-broker returns event with this `eventDate` field instead of `period` like `analytics` do huh? Oh well, that's too big of an issue to deal with right now 😝